packages/flutter/lib/src/cupertino/context_menu.dart

```diff
@@ -545,10 +566,11 @@ class _CupertinoContextMenuState extends State<CupertinoContextMenu> with Ticker
     });
 
     final Rect childRect = _getRect(_childGlobalKey);
+    _scaleFactor = _computeScaleFactor(context, childRect);
```

Comment:
Nit: I think it would be better to pass the padding in here rather than the context, so that the static method has no side effects. Maybe that's just me that cares about that though.

Reply:
Also passed the size too.